examples/ReorderExample.js

```diff
@@ -65,42 +64,59 @@ class ReorderExample extends React.Component {
       columnOrder.splice(index, 0, event.reorderColumn);
     } else {
       if (fixedColumns.indexOf(event.reorderColumn) !== -1) {
-        columnOrder.splice(fixedColumns.length - 1, 0, event.reorderColumn)
+        columnOrder.splice(fixedColumns.length - 1, 0, event.reorderColumn);
       } else {
         columnOrder.push(event.reorderColumn);
       }
     }
     this.setState({
-      columnOrder: columnOrder
+      columnOrder: columnOrder,
     });
-  }
+  };
 
-  render() {
-    var {dataList} = this.state;
+  toggleCellsRecycling = (enable, columnKey) => {
+    this.setState({
+      recycling: {
+        [columnKey]: enable,
+      },
+    });
+  };
 
+  render() {
+    var { dataList, recycling } = this.state;
+    var onColumnReorderEndCallback = this._onColumnReorderEndCallback;
+    const toggleCellsRecycling = this.toggleCellsRecycling;
```

Comment:
rename to onColumnReorderStart and just inline this to the cell

Reply:
Done